ludwig/schema/metadata/configs/llm.yaml

```diff
@@ -119,3 +119,17 @@ adapter:
     adapter_layers:
       ui_display_name: Adapter Layers
       expected_impact: 3
+quantization:
+  _oneOf:
+    allOf:
```

Comment:
ultimately these are just meta properties that you can name whatever you want, but strictly speaking there is no `allOf` schema for `quantization` - it's just that the entire thing is disabled or you specify the full object.

Reply:
Interesting, was just going off the pattern for `adapter`. What should it be changed to?

Reply:
I would call it "enabled" or "object" or "non-null" or something like that. These middle-management words don't semantically *mean* anything - they're just a reference for the coder to be able to tell a schema where to pull the parameter metadata from.

So you haven't done this yet, but if I looked at the code for `jsonschema_type_mapping` and saw this:
```
{
  "oneOf": {
    {
      "title": "enabled"
      ...
      "parameter_metadata": LLM_METADATA["quantization"]["_oneOf"]["allOf"]
    },
    ...
  }
  "title": "quantization",
  ...
  "parameter_metadata":  LLM_METADATA["quantization"]["_meta"]
}
```

I would get slightly confused - I would think to myself "allOf? I don't see an allOf in this code".